spec/models/need_spec.rb

```diff
@@ -14,8 +14,8 @@
   end
 
   describe 'scopes' do
-    let!(:uncompleted_need) { create :need }
-    let!(:completed_need) { create :need, :completed }
+    let!(:uncompleted_need) { create :need, status: :to_do }
```

Comment:
Should this actually be any state that in't complete?

Reply:
Yes, and we start from `to_do`. This is why I put this one.